### ISSUE TITLE
Audio not encoding

### ISSUE THREAD

**Issue by aeryx** [mps-youtube/mps-youtube#486]
I've installed ffmpeg with mp3lame, configured mpsyt to encode in mp3 256k and edited the text config file to point to ffmpeg binary, but all dowloads are still saved with the source encoding. What am I missing?

**Comment by speedemonV12** [comment]
I am also experiencing this problem. Both on Arch and Ubuntu. Any help appreciated. Thanks!

**Comment by speedemonV12** [comment]
Update: Encoder seems to work when downloading the m4a source. That seems to be the only one that mps automatically converts to mp3. The rest all download in their source encoding and not being converted to mp3

**Comment by spcmd** [comment]
@speedemonV12 I had to add webm to the "valid for:" lines in the transcode file. See [this comment](https://github.com/mps-youtube/mps-youtube/issues/389#issuecomment-164239705) from #389

**Comment by ZaxonXP** [comment]
Indeed, when the source is m4a AND the "download_command" is set to "none" then the conversion take place. Otherwise the file is not converted. It would be nice if the conversion is also done when the external downloader is in use.
